doc/source/cluster/cli.rst

```diff
@@ -56,4 +58,4 @@ This section contains commands for managing Ray clusters.
 
 .. click:: ray.scripts.scripts:monitor
    :prog: ray monitor
-   :show-nested:
\ No newline at end of file
+   :show-nested:
```

Comment:
I don't know why this line shows up in the diff even though it's unchanged.  Maybe a bug in github.